projects/packages/stats-admin/src/class-rest-controller.php

```diff
@@ -606,9 +606,15 @@ public function get_single_post( $req ) {
 
 		// It shouldn't be a problem because only title and ID are exposed.
```

Comment:
I'm curious about this comment. Do you know what it's referring to? We are no longer only exposing the title and ID here, and have added additional information. I've read through the code and can't see what "It shouldn't be a problem" might mean in this context, or how exposing additional data could affect that?

Reply:
Good catch @annacmc! The comment here is out dated. Addressed in 79d2478c97